openfe/protocols/openmm_afe/base.py

```diff
@@ -406,14 +407,16 @@ def _get_lambda_schedule(
           LambdaProtocol
         """
         lambdas = dict()
-        n_elec = settings['alchemical_settings'].lambda_elec_windows
-        n_vdw = settings['alchemical_settings'].lambda_vdw_windows + 1
-        lambdas['lambda_electrostatics'] = np.concatenate(
-                [np.linspace(1, 0, n_elec), np.linspace(0, 0, n_vdw)[1:]]
-        )
-        lambdas['lambda_sterics'] = np.concatenate(
-                [np.linspace(1, 1, n_elec), np.linspace(1, 0, n_vdw)[1:]]
-        )
+
+        lambda_elec = settings['lambda_settings'].lambda_elec
+        lambda_vdw = settings['lambda_settings'].lambda_vdw
+
+        # Reverse lambda schedule since in AbsoluteAlchemicalFactory 1
+        # means fully interacting, not stateB
+        lambda_elec = [1-x for x in lambda_elec]
```

Comment:
TODO: need to check if upstream expects any iterable or a numpy array

Reply:
Also python floats might be a bit more succeptible to float truncation, i.e.

```
[1.0,
 0.9,
 0.8,
 0.7,
 0.6,
 0.5,
 0.4,
 0.30000000000000004,
 0.19999999999999996,
 0.09999999999999998,
 0.0]
```

Reply:
I'm not sure numpy will have better float handling here, it's just a fundamental fact about floating points, you'd instead need something like this: https://docs.python.org/3/library/decimal.html 

If this is just getting used to scale numbers, then floats are fine, if it needs to look nice then we can use decimal

Reply:
As per today's call: upstream expectation is a dictionary of list